lsp-headerline.el

```diff
@@ -161,11 +161,16 @@ Switch to current mouse interacting window before doing BODY."
                                        directory-display-string)
                                directory-display-string))
 
+(declare-function evil-set-jump "ext:evil-jumps")
+
 (lsp-defun lsp-headerline--symbol-with-action ((symbol &as &DocumentSymbol :name) symbol-display-string)
   "Build action for SYMBOL and SYMBOL-STRING."
   (lsp-headerline--with-action (let ((map (make-sparse-keymap)))
                                  (define-key map [header-line mouse-1]
                                    (lsp-headerline--make-mouse-handler
+                                     (when (and (require 'evil nil t)
```

Comment:
seems like the `require` part is not needed here.

Reply:
Fixed.